Notification.js

```diff
@@ -26,7 +24,7 @@ class Notification extends Component {
     };
   }
 
-  show(title = '', message = '', onPress = null) {
+  show({ title, message, onPress, icon, vibrate } = { title: '', message: '', onPress: null, icon: null, vibrate: true }) {
```

Comment:
Could you let me know why you changed this please 😄

Reply:
Because the parameters are optional, the best way to pass them is in this format.

```js
// bad
this._notification.show(
  'My Title',
  '',
  () => {
    // handle click notification
  },
});

// good
this._notification.show({
  title: 'My Title',
  onPress: () => {
    // handle click notification
  },
});
```

Reply:
Right! Gotcha, I didn't see that you'd changed the input params for the `show` function. Could you also please add a note in README regarding this change as this is a breaking change and needs a large version bump as per [SemVer](http://semver.org/)